www/src/components/LayoutHeader/Header.js

```diff
@@ -161,6 +161,7 @@ const Header = ({location}) => (
             css={{
               appearance: 'none',
               background: 'transparent',
+              outline: 'none',
```

Comment:
I think we should revert this until we have a custom focus style.

Reply:
there will be a input indicator there when focused, the live jsx editor focus in the same way

Reply:
Without the `outline` style, there's no visual indicator of focus on the desktop site. There is on the mobile-sized layout, but `outline` is already suppressed there.

Anyway. Thanks for reverting this bit. 😄